### PR TITLE
Require Node.js 6

### PR DESCRIPTION
**Pull request by SamVerschueren**
The usual.

- `.npmrc`
- No hard line wrapping in `license` file
- Spaces to tabs in `package.json`
- Add Node.js 8 and Node.js 10 to `.travis.yml`

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -1,29 +1,29 @@
 import test from 'ava';
 import Bluebird from 'bluebird';
-import m from './';
+import isPromise from '.';
```
Comment (SamVerschueren) [inline test.js:3]: You told me that we should go for the module name instead. If you want me to revert it, let me know :).

Also, do you want me to fix the versions of AVA and XO?

Comment (sindresorhus) [reply test.js:3]: > Also, do you want me to fix the versions of AVA and XO?

👍

**Comment by sindresorhus** [comment]
You can also drop Node.js 4 support if you want. I do that now anytime I touch a module.

**Comment by SamVerschueren** [comment]
Done

**Pull request merged**